packages/ui/src/ToolTip/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import { createPortal } from "react-dom";
+
+interface Properties {
+  text: string;
+  children: React.ReactNode;
+}
```

Comment:
It would be great, if dev can easily set position like top, right, bottom and left,
also support style and className for tooltip through properties.

Reply:
It would be great if we could position tooltip just by sting values like top, right, bottom, and left,
for other we can provide style props.